lib/tasks/active_stash.rake

```diff
@@ -27,7 +27,23 @@ namespace :active_stash do
     info("")
     info("")
     info("")
-    info("Opening your browser to complete signup.......")
+    info("")
+    info("")
+    info("You are being redirected to https://console.cipherstash.com/?signup=active-stash to complete your signup..")
+    info("")
+    info("")
+    info("NEXT STEPS:")
+    info("")
+    info("1. Signup with your github account or email.")
```

Comment:
```suggestion
    info("1. Sign up with your GitHub account or email.")
```

Minor nits:
* I think it's "sign up" as a verb and "signup" as a noun
* Capitalisation for GitHub

Reply:
Thanks @CDThomas , i'll do a quick follow up. Sorry for being so trigger happy 😆